lib/cc/engine/analyzers/javascript/main.rb

```diff
@@ -29,7 +29,7 @@ def run
           end
 
           def mass_threshold
```

Comment:
Yep, I started thinking the same thing and was going to look into it after this PR is good to go.